sfge/src/main/java/com/salesforce/graph/build/AbstractApexVertexBuilder.java

```diff
@@ -102,6 +117,8 @@ private void buildVertices(JorjeNode node, Vertex vNodeParam, String fileName) {
     private final void afterInsert(GraphTraversalSource g, JorjeNode node, Vertex vNode) {
         if (node.getLabel().equals(ASTConstants.NodeType.METHOD)) {
             MethodPathBuilderVisitor.apply(g, vNode);
+        } else if (ROOT_VERTICES.contains(node.getLabel())) {
+            StaticBlockUtil.createSyntheticStaticBlockInvocation(g, vNode);
```

Comment:
If we are at the end of handling a root (class/interface/trigger etc) node, check if any synthetic static block methods were created earlier. If yes, add a synthetic static block invoker method to invoke the static block methods.